test/DebugInfo/Inputs/type-reconstr-names.txt

```diff
@@ -3,7 +3,6 @@ $SytD ---> ()
 $Ss5Int32VD ---> Int32
 $S4blah4mainyyF8PatatinoL_VMa ---> Can't resolve type of $S4blah4mainyyF8PatatinoL_VMa
 $Ss10CollectionP7Element ---> Can't resolve type of $Ss10CollectionP7Element
-$Ss15ContiguousArrayV9formIndex5afterySiz_tFSS_Tg5 ---> (inout Int) -> ()
```

Comment:
No, this needs to continue to work since its still valid to have *values* whose types are function types with inout parameters.